database/seeders/DatabaseSeeder.php

```diff
@@ -14,17 +14,11 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-        if (App::environment('production')) {
-            $this->call(AdminSeeder::class);
-        } else {
-            $this->call([
-                UserSeeder::class,
-                AdminSeeder::class,
-                DonationSeeder::class,
-                DonationRequestSeeder::class,
-                PetitionSeeder::class,
-            ]);  
-        }
-
+        $this->call([
+            UserSeeder::class,
+            DonationSeeder::class,
+            DonationRequestSeeder::class,
+            PetitionSeeder::class,
+        ]);
     }
```

Comment:
We should not delete this, because the seeds are going to be run on the development environments

Reply:
done